src/main/java/core/basesyntax/suppliers/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.suppliers;
+
+import core.basesyntax.enums.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        return Color.values()[new Random().nextInt(Color.values().length)].name();
```

Comment:
Here, your random instance is created every time the method is called. random - is our dependency, let's initialize dependencies using a constructor or directly create a new instance in the class fields.